python/test/t_ExpertMixture_std.py

```diff
@@ -1,38 +1,41 @@
 #! /usr/bin/env python
 
 from __future__ import print_function
-from openturns import *
+import openturns as ot
 
-TESTPREAMBLE()
-RandomGenerator.SetSeed(0)
-
-
-aCollection = DistributionCollection(0)
-R = CorrelationMatrix(2)
+R = ot.CorrelationMatrix(2)
 R[0, 1] = -0.99
-aCollection.add(Normal([-1.0, 1.0], [1.0, 1.0], R))
+d1 = ot.Normal([-1.0, 1.0], [1.0, 1.0], R)
 R[0, 1] = 0.99
-aCollection.add(Normal([1.0, 1.0], [1.0, 1.0], R))
+d2 = ot.Normal([1.0, 1.0], [1.0, 1.0], R)
+distribution = ot.Mixture([d1, d2], [1.0]*2)
+classifier = ot.MixtureClassifier(distribution)
+f1 = ot.SymbolicFunction(['x'], ['-x'])
+f2 = ot.SymbolicFunction(['x'], ['x'])
+experts = ot.Basis([f1, f2])
+moe = ot.ExpertMixture(experts, classifier)
+moeNMF = ot.Function(moe)
 
-# Instanciate one distribution object
-distribution = Mixture(aCollection, Point(aCollection.getSize(), 1.0))
+print('Mixture of experts=', moe)
 
-# Create a mixture classifier
-classifier = MixtureClassifier(distribution)
+# Evaluate the mixture of experts on some points
+for i in range(2):
+    p = [-0.3 + 0.8 * i / 4.0]
+    print('moe   ( %.6g )=' % p[0], moe(p))
+    print('moeNMF( %.6g )=' % p[0], moeNMF(p))
 
-# Create local experts
-experts = Basis()
-experts.add(SymbolicFunction("x", "-x"))
-experts.add(SymbolicFunction("x", "x"))
 
-# Create a mixture of experts
-moe = ExpertMixture(experts, classifier)
-moeNMF = Function(moe)
+# non-supervised mode (2d)
+f1 = ot.SymbolicFunction(['x1', 'x2'], ['-8'])
```

Comment:
As we are introducing the non-supervised mode it should *correctly* be tested :
 - at least 2 different use case
 - on a sample and not only two points